### PR TITLE
Include instructions for oca_dependencies.txt

### PR DESCRIPTION
**Pull request by Yajo**
These instructions are copied from https://github.com/OCA/maintainer-quality-tools/blob/67d3880b085a99d820e5a65cc7354dae95d1b4b0/sample_files/oca_dependencies.txt. This is a better place to put them.

### CODE REVIEW

File: website/Contribution/CONTRIBUTING.rst [hunk 1]
```diff
@@ -303,6 +303,32 @@ As specified in [Repositories](#repositories), you should also define
...
+    sale-workflow https://github.com/OCA/sale-workflow
+
+To provide both the URL and a branch, use::
+
+    sale-workflow https://github.com/OCA/sale-workflow branchname
+
+To use a specific commit version, set the branch (required) and the
+commit SHA to select::
+
+    sale-workflow https://github.com/OCA/sale-workflow branchname f848e37
```
Comment (pedrobaeza) [inline website/Contribution/CONTRIBUTING.rst:330]: `branchname`?

Comment (Yajo) [reply website/Contribution/CONTRIBUTING.rst:330]: Yes, that's how it was redacted... :shrug:

Comment (pedrobaeza) [reply website/Contribution/CONTRIBUTING.rst:330]: @moylop260  can you confirm this is correct?

Comment (Yajo) [reply website/Contribution/CONTRIBUTING.rst:330]: This is already OCA code, I'm just moving it elsewhere.

I don't think we really need to review it again, apart from rst syntax etc. Can't we assume this was reviewed when introduced for the 1st time?

I say just for fast-tracking this.

Comment (moylop260) [reply website/Contribution/CONTRIBUTING.rst:330]: Yes, it is correct.
 - https://github.com/OCA/maintainer-quality-tools/blob/67d3880b085a99d820e5a65cc7354dae95d1b4b0/sample_files/oca_dependencies.txt#L19

Comment (pedrobaeza) [reply website/Contribution/CONTRIBUTING.rst:330]: OK, thanks

**Comment by pedrobaeza** [comment]
I'm not able to push here. @sbidoul @dreispt can you give me permission or merge?

**Pull request merged**